build/Platform.targets

```diff
@@ -0,0 +1,156 @@
+      Targets="Pack"
+      Properties="Configuration=$(Configuration);DotNetRestoreSourcePropsPath=$(GeneratedRestoreSourcesPropsPath);AspNetUniverseBuildOffline=true" />
+
+    <!-- Copy to output directory -->
+    <ItemGroup>
+      <AllMetapackageNupkgFile Include="$(MetapackageWorkDirectory)**\*.nupkg" />
+    </ItemGroup>
+
+    <Copy SourceFiles="@(AllMetapackageNupkgFile)" DestinationFolder="$(BuildDir)" />
+  </Target>
+
+  <Target Name="GetLatestCommitHash">
+    <!-- Get the latest commit hash -->
+    <Exec Command="git rev-parse HEAD 2>&amp;1" StandardOutputImportance="Low" IgnoreExitCode="true" IgnoreStandardErrorWarningFormat="true" ConsoleToMSBuild="true">
+      <Output TaskParameter="ConsoleOutput" PropertyName="LatestCommit" />
+      <Output TaskParameter="ExitCode" PropertyName="LatestCommitExitCode" />
+    </Exec>
+    <!-- We shouldn't fail the build if we can't retreive the commit hash, so in this case just set it to N/A -->
+    <PropertyGroup Condition="'$(LatestCommitExitCode)'!='0'">
+      <LatestCommit>N/A</LatestCommit>
+    </PropertyGroup>
+  </Target>
+
+  <Target Name="BuildPlatform" DependsOnTargets="GetLatestCommitHash">
+    <Error Text="Please specify the RID for the aspnetcore platform via PlatformRID: {win7-x64|win7-x86|osx-x64|linux-x64}." Condition="'$(PlatformRID)' == ''"/>
+
+    <!-- Clear working directory -->
+    <RemoveDir Directories="$(_WorkRoot)" />
+
+    <PropertyGroup>
+      <PlatformWorkDirectory>$(_WorkRoot)Platform\</PlatformWorkDirectory>
+      <PlatformOutputDirectory>$(_WorkRoot)Publish\</PlatformOutputDirectory>
+      <PlatformPublishDirectory>$(PlatformOutputDirectory)shared\Microsoft.AspNetCore.All\$(PackageVersion)\</PlatformPublishDirectory>
+
+      <_PlatformProps>Configuration=$(Configuration);RuntimeIdentifier=$(PlatformRID)</_PlatformProps>
+      <_PlatformProps>$(_PlatformProps);RuntimeFrameworkVersion=$(MicrosoftNETCoreApp20PackageVersion)</_PlatformProps>
+      <_PlatformProps>$(_PlatformProps);DotNetRestoreSourcePropsPath=$(GeneratedRestoreSourcesPropsPath)</_PlatformProps>
+      <_PlatformProps>$(_PlatformProps);AspNetUniverseBuildOffline=true</_PlatformProps>
+    </PropertyGroup>
+
+    <!-- Move to working dir -->
+    <ItemGroup>
+      <PlatformFiles Include="$(_TemplatesDir)Platform\**\*" />
+    </ItemGroup>
+
+    <Copy SourceFiles="@(PlatformFiles)" DestinationFolder="$(PlatformWorkDirectory)\%(RecursiveDir)" />
+    <Copy SourceFiles="$(_SrcDir)Directory.Build.props" DestinationFolder="$(_WorkRoot)" />
+
+    <!-- Set _Target=Restore so the project will be re-evaluated to include Internal.AspNetCore.Sdk MSBuild properties on the next step. -->
+    <MSBuild Projects="$(PlatformWorkDirectory)Platform.csproj"
+      Targets="Restore"
+      Properties="$(_PlatformProps);MicrosoftAspNetCoreAllVersion=$(PackageVersion);_Target=Restore" />
+
+    <!-- Publish -->
+    <MSBuild Projects="$(PlatformWorkDirectory)Platform.csproj"
+      Targets="Publish"
```

Comment:
Still verifying a few properties with the dotnet teams. The produced output is able to run built and published apps but I see a few anomalies in the deps files and some versioning oddities. I'll file followup issues.